src/Services/Cloudflare.php

```diff
@@ -45,10 +45,14 @@ public static function getDomainStatistics(string $authKey, string $authEmail, a
                 return [
                     $key => [
                         'requests' => $item->sum(static function (array $item) {
-                            return $item['httpRequests1dGroups'][0]['sum']['requests'] ?: 0;
+                            return !empty($item['httpRequests1dGroups'][0]['sum']['requests']) 
+                                ? $item['httpRequests1dGroups'][0]['sum']['requests'] 
+                                : 0;
```

Comment:
LGTM. Would using null coalescing be neater here? 🤔
```suggestion
                            return $item['httpRequests1dGroups'][0]['sum']['requests'] ?? 0;
```

Reply:
For sure! thanks